static/js/input.js

```diff
@@ -632,4 +621,4 @@ function parseContigTitle(line, format) {
     return [id, coverage];
   }
   return null;
-}
+}
```

Comment:
GitHub is reminding you that there should be an empty line at the end of each script file.